runway/data_types.py

```diff
@@ -646,3 +644,88 @@ def to_dict(self):
         ret = super(boolean, self).to_dict()
         ret['default'] = self.default
         return ret
+
+
+class point(BaseType):
+    """A point data type representing a specific location in an image. 
+    It accepts two normalized floating point numbers [x, y] between 0 and 1,
+    where [0, 0] represents the top-left corner and [1, 1] the bottom-right corner of an image.
+    
+    .. code-block:: python
+
+        import runway
+        from runway.data_types import image, point
+
+        @runway.command('detect_gaze', inputs={'image': image()}, outputs={'gaze_location': point()})
+        def detect_gaze(model, inputs):
+            result = model.run(inputs['image'])
+            return {'gaze_location': result}
+
+    :param description: A description of this variable and how it's used in the model,
+        defaults to None
+    :type description: string, optional
+    """
+    def __init__(self, description=None):
+        super(point, self).__init__('point', description=description)
+    
+    def validate(self, value):
+        if len(value) == 2:
+            if not all([0 <= item <= 1 for item in value]):
+                raise InvalidArgumentError(self.name, 'Point coordinates must be between 0 and 1')
+        else:
+            raise InvalidArgumentError(self.name, 'Value must be of length 2')
+        
+    def deserialize(self, value):
+        self.validate(value)
+        return value
+    
+    def serialize(self, value):
+        value = [try_cast_np_scalar(item) for item in value]
+        self.validate(value)
+        return value
+
+
+class bounding_box(BaseType):
+    """An bounding box data type, representing a rectangular region in an image.
+    It accepts four normalized floating point numbers [xmin, ymin, xmax, ymax] between 0 and 1,
+    where [xmin, xmax] is the top-left corner of the rectangle and [xmax, ymax] is the bottom-right corner.
+    
+    .. code-block:: python
+
+        import runway
+        from runway.data_types import image, point
+
+        @runway.command('detect_face', inputs={'image': image()}, outputs={'face_bbox': bounding_box})
+        def detect_faze(model, inputs):
+            result = model.run(inputs['image'])
+            return {'face_bbox': result}
+
+    :param description: A description of this variable and how it's used in the model,
+        defaults to None
+    :type description: string, optional
+    """
+    def __init__(self, description=None):
```

Comment:
Are we choosing to divorce an optional `label` from a `bounding_box` and associate them instead as similarly positioned keys in a higher data structure to render in the UI?

```python
{
  'labels': ['human', 'dog', None],
  'boxes': [..., ..., ...]
}
```